tests/Constraints/OfPropertiesTest.php

```diff
@@ -72,22 +72,22 @@ public function getInvalidTests()
                 null,
                 array(
                     array(
-                        "property"   => "prop2",
-                        "pointer"    => "/prop2",
-                        "message"    => "Array value found, but a string is required",
-                        "constraint" => "type",
+                        'property' => 'prop2',
```

Comment:
That's a fair call. @bighappyface & @shmax - what are your thoughts on enabling this globally?

If global is undesirable for some reason, would it be possible to enable it just for the testsuite? That's where most of the array stuff that needs aligning is located anyway.